poi/src/main/java/org/apache/poi/ss/formula/functions/Baseifs.java

```diff
@@ -40,7 +40,17 @@
      * @return true if there should be a range argument before the criteria pairs
      */
     protected abstract boolean hasInitialRange();
-        
+  
+    /**
+     * Implements the details of a specific aggregation functions
+     */
+    protected static interface Aggregator {
+    	void addValue(ValueEval d);
```

Comment:
likewise - the POI standard is 4 space indents

Reply:
Thank you very much for reviewing and the comments. I fixed the whitespace and made the tests pass. Should I issue a new PR?

Reply:
just commit the extra changes to your trunk branch and this PR will be automatically updated